assignments/closure.js

```diff
@@ -1,20 +1,51 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+let array = [1,2,3,4,5];
```

Comment:
You clearly have a good understanding of closure. Keep up the good work.  I enjoy the detailed console logs as well.